web/js/components/timeline/timeline-data/timeline-data.js

```diff
@@ -263,6 +265,37 @@ class TimelineData extends Component {
     }
   }
 
+  stopPropagation = (e) => {
+    e.nativeEvent.stopImmediatePropagation();
+    e.stopPropagation();
+    e.preventDefault();
+  }
+
+  /**
+  * @desc render info button for data panel coverage info modal
+  * @returns {DOM Object}
+  */
+  renderInfoButton = () => {
+    const { onInfoClick } = this.props;
+    const layerInfoBtnId = 'data-panel-info-button';
+    const layerInfoBtnTitle = 'Data Coverage Panel Information';
+
+    return (
+      <a
+        id={layerInfoBtnId}
+        aria-label={layerInfoBtnTitle}
+        className={layerInfoBtnId}
+        onMouseDown={this.stopPropagation}
+        onClick={() => onInfoClick()}
+      >
+        <UncontrolledTooltip placement="top" target={layerInfoBtnId}>
```

Comment:
Having a tooltip show on this icon felt a little disorienting to me.  When I see an info icon that I can click I have some pretty clear expectations of what will happen so I'm not sure that it's needed.
![Screen Shot 2021-01-29 at 8 25 16 AM](https://user-images.githubusercontent.com/1480000/106280863-719a4f80-620c-11eb-9844-49611d2c81ac.png)